solutions/swb-ui/ui/package.json

```diff
@@ -21,7 +21,7 @@
     "build:test": "rushx build && rushx test",
     "check-license-header": "license-check-and-add check -f license-add-config.json",
     "depcheck": "depcheck",
-    "deploy-ui-and-api": "bash ./scripts/deploy.sh",
+    "deploy-ui-and-api": "bash ./scripts/image-deploy.sh $BRANCH && bash ./scripts/deploy.sh",
```

Comment:
How is `$BRANCH` resolved here, and could we pin it to a specific branch (like develop) till the release?

Reply:
nvm, I see it's set to develop if not passed

Reply:
`$BRANCH` gets passed in as an env var like STAGE; if none is passed in, the image-deploy.sh script will default to use develop.